webrtc-stats.html

```diff
@@ -1491,8 +1492,29 @@ <h2>
                   interoperability issues.
                 </p>
                 <p class="fingerprint">
-                  If too much information is given here, it increases the fingerprint surface.
-                  Since it is only given for active tracks, the incremental exposure is small.
+                  As this field is a fingerprinting vector, it MUST only be
+                  exposed to contexts that the user interacted with in a deep
+                  manner, for instance if <a href="https://w3c.github.io/mediacapture-main/#context-capturing-state">
+                  context capturing state</a> returns true.
```

Comment:
This sentence is repeated a few times, let me know if I should move this to a hardware fingerprinting section and point to it to avoid repeated information